conanfile.py

```diff
@@ -320,6 +324,32 @@ def build_configure(self):
             if self.settings.os == "Windows":
                 args.append('--enable-libmfx' if self.options.qsv else '--disable-libmfx')
 
+            if tools.cross_building(self.settings):
+                args.append('--enable-cross-compile')
+                if self.settings.os == "Android":
+                    args.append('--target-os=android')
+                    if "android_ndk_installer" in self.deps_env_info:
+                        ndk_env_info = self.deps_env_info["android_ndk_installer"]
+                        cc = ndk_env_info.CC
```

Comment:
I think it's redundant, as env variables from `android_ndk_installer` are in os.environ, right?